app/test/model/post.test.js

```diff
@@ -72,3 +72,19 @@ test('add post and use isSlugExist function to find out if the slug was unique',
     await postModel.deletePost(post1);
     expect(await postModel.isSlugExist(post1)).toEqual(false);
 })
+
+test('check getting all posts', async () => {
+    let newPost1 = new postModel.PostEntity(null, 'first title', 'slug-1', 'content 1');
+    await postModel.insertPost(newPost1);
+
+    let allPosts = [];
+    allPosts = await postModel.getPosts();
+
+    let amount = 0;
+    for (let i = 0; i < allPosts.length; i++) {
+        amount = +i;
+    }
+    expect(await allPosts[amount]).toEqual(newPost1);
```

Comment:
Tutaj sprawdzasz czy ostatnim postem jest ten nowo dodany co niekoniecznie zawsze musi być prawdą. Mogłabyś to zrobić tak, bez tej pętli

```
expect(await allPosts[allPosts.length - 1]).toEqual(newPost1);
```

Oczywiście nie o to nam chodzi. Po prostu sprawdź czy w tablicy `allPosts` znajduje się element `newPost1`

Reply:
robiłam to `await allPosts[allPosts.length - 1]` nie działało.

Reply:
zrobione